### PR TITLE
Bulk Delete https://github.com/gterrono/houston/issues/260

### PR DESCRIPTION
**Pull request by artwells**
OP closed the issue, but I wanted it.

### CODE REVIEW

**Comment by gterrono** [comment]
:+1: Looks good to me.

I still can't test things for the moment, so can you merge and check @AlexeyMK?

File: server/methods.coffee [hunk 1]
```diff
@@ -39,4 +39,12 @@ Houston._setup_collection_methods = (collection) ->
       id = collection.findOne(new Meteor.Collection.ObjectID(id))
       collection.remove(id)
 
+  methods[Houston._houstonize "#{name}_deleteAll"] = require_admin () ->
```
Comment (AlexeyMK) [inline server/methods.coffee]: nit: all of this whitespace is not necessary

File: client/collection_view.coffee [hunk 2]
```diff
@@ -166,6 +166,12 @@ Template._houston_collection_view.events
     if confirm("Are you sure you want to delete the document with _id #{id}?")
       Houston._call("#{@collection}_delete", id)
 
+  'click .houston-delete-all': (e) ->
+    e.preventDefault()
+    name = $(e.currentTarget).data('name')
+    if confirm("Are you sure you want to delete all document from #{name}?")
```
Comment (AlexeyMK) [inline client/collection_view.coffee]: nit: `all the documents`, not `all document`

**Comment by AlexeyMK** [comment]
a few minor points, but seems largely reasonable to me.

**Comment by AlexeyMK** [comment]
There were some style issues with the buttons, but I ended up fixing myself and shipping. Thanks @artwells!

**Comment by artwells** [comment]
Thanks @AlexeyMK.  I'm using houston to extend https://atmospherejs.com/artwells/queue, and this really helps particularly with the logs.

**Pull request merged**